INSTALL.md

```diff
@@ -46,84 +35,157 @@ platforms.
+We can now compile and install `google-cloud-cpp`. Note that we use
+`pkg-config` to discover the options for gRPC and protobuf:
 
 ```bash
-cd $HOME/google-cloud-cpp # or wherever you have extracted google-cloud-cpp
+cd /home/build/google-cloud-cpp
 cmake -H. -Bbuild-output \
     -DGOOGLE_CLOUD_CPP_DEPENDENCY_PROVIDER=package \
     -DGOOGLE_CLOUD_CPP_PROTOBUF_PROVIDER=pkg-config \
     -DGOOGLE_CLOUD_CPP_GRPC_PROVIDER=pkg-config \
     -DGOOGLE_CLOUD_CPP_GMOCK_PROVIDER=external
 cmake --build build-output -- -j $(nproc)
-cd build-output
+cd /home/build/google-cloud-cpp/build-output
 ctest --output-on-failure
-cmake --build . --target install
+sudo cmake --build . --target install
 ```
 
-### Ubuntu (Bionic Beaver)
 
-First install the development tools, libcurl, and OpenSSL.
+### OpenSUSE (Tumbleweed)
+
+Install the minimal development tools:
 
 ```bash
-sudo apt update
+sudo zypper refresh && \
+sudo zypper install -y cmake gcc gcc-c++ git libcurl-devel libopenssl-devel make
+```
+
+OpenSUSE:tumbleweed provides packages for gRPC, libcurl, and protobuf, and the
+versions of these packages are recent enough to support the Google Cloud
+Platform proto files.
+
+```bash
+sudo zypper refresh && \
+sudo zypper install -y grpc-devel libcurl-devel pkg-config tar wget
+```
+
+#### crc32c
+
+There is no OpenSUSE package for this library. To install it, use:
+
+```bash
+cd /var/tmp/build
+wget -q https://github.com/google/crc32c/archive/1.0.6.tar.gz
+tar -xf 1.0.6.tar.gz
+cd /var/tmp/build/crc32c-1.0.6
+cmake \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DBUILD_SHARED_LIBS=yes \
+      -DCRC32C_BUILD_TESTS=OFF \
+      -DCRC32C_BUILD_BENCHMARKS=OFF \
+      -DCRC32C_USE_GLOG=OFF \
+      -H. -B.build/crc32c
+sudo cmake --build .build/crc32c --target install -- -j $(nproc)
+sudo ldconfig
+```
+
+#### google-cloud-cpp
+
+We can now compile and install `google-cloud-cpp`. Note that we use
+`pkg-config` to discover the options for gRPC and protobuf:
+
+```bash
+cd /home/build/google-cloud-cpp
+cmake -H. -Bbuild-output \
+    -DGOOGLE_CLOUD_CPP_DEPENDENCY_PROVIDER=package \
+    -DGOOGLE_CLOUD_CPP_PROTOBUF_PROVIDER=pkg-config \
+    -DGOOGLE_CLOUD_CPP_GRPC_PROVIDER=pkg-config \
+    -DGOOGLE_CLOUD_CPP_GMOCK_PROVIDER=external
+cmake --build build-output -- -j $(nproc)
+cd /home/build/google-cloud-cpp/build-output
+ctest --output-on-failure
+sudo cmake --build . --target install
+```
+
+
+### Ubuntu (18.04 - Bionic Beaver)
+
+Install the minimal development tools:
+
+```bash
+sudo apt update && \
 sudo apt install -y build-essential cmake git gcc g++ cmake \
         libc-ares-dev libc-ares2 libcurl4-openssl-dev libssl-dev make \
         pkg-config tar wget zlib1g-dev
 ```
 
 #### crc32c
- 
+
 There is no Ubuntu package for this library. To install it use:
 
 ```bash
 cd /var/tmp/build
 wget -q https://github.com/google/crc32c/archive/1.0.6.tar.gz
 tar -xf 1.0.6.tar.gz
-cd build/crc32c-1.0.6
+cd /var/tmp/build/crc32c-1.0.6
```

Comment:
Instructions now build at /var/tmp/ I think this makes sense for Docker but may not for end users.

Reply:
Changed to `$HOME/Downloads`, that directory may not exist, but it is kind of obvious how to create it.